docs/platform_management_plan/project_management.rst

```diff
@@ -181,4 +181,219 @@ meetings, where such topics are discussed and decided, see `Steering committees`
+  a child of any *Saga*. *Saga* should be the only way for grouping *Epics* together.
+  Grouping standalone *Epics* with labels is not something, what we encourage you to do.
+* *Story* *GitHub Issue* of type *Story* is the lowest planning granularity and represents
+  concrete task, that should be done, e.g. by a developer. Normally *Stories* are
+  grouped together in an Epic. In some cases a *Story* can exist as a standalone *GitHub issue*.
+  Grouping standalone *Stories* with labels is not something, what we encourage you to do.
+* *Bug* *GitHub Issue* of type *Bug* is used to report any kind of problems. It can be
+  a standalone *GitHub Issue* or can be a sub-issue of an *Epic* or a *Saga*.
+  It is also ok to use labels to group multiple *Bugs* that are related to the same topic.
+
+Main *SCORE* project defines templates for every type of *GitHub Issues*
+to ensure, that every ticket has all necessary information.
+
+For a better structuring of the *GitHub Issues*, we use a beta
+`sub-issue feature <https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/adding-sub-issues>`_,
+that should be officially released in the beginning of 2025.
+*Sub-issue feature* allows to create a "parent-child" relationship between *GitHub Issues*.
+That allows better structuring of the project and helps to keep *GitHub Issues*, that
+are related to the same topic, together.
+
+.. image:: _assets/sub_issues.png
+    :width: 600
+    :alt: Sub issues overview
+    :align: center
+
+
+Traceability
+^^^^^^^^^^^^
+To achieve a better traceability it is highly recommended to link all *PRs* to the corresponding
+*GitHub Issues*. If done properly, you will be able to see for every *GitHub Issue*
+all relevant source code changes. Normally *PRs* reference *GitHub issues* of type *Story*
+or of type *Bug*. How to link *PRs* to *GitHub Issues* is described in more details in this
+`guide <https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/linking-a-pull-request-to-an-issue>`_.
+
+.. image:: _assets/traceability.png
+    :width: 300
+    :alt: Traceability overview
+    :align: center
+
+GitHub Projects
+^^^^^^^^^^^^^^^
+*GitHub Projects* is a very powerful tool that allows creation of various views on
+the status of the project, helps to plan the work and to monitor the current progress.
+In particular, *GitHub Project* allows to extend *GitHub Issues* with following information:
+
+* objective
+* dependencies on other activities or information
+* responsible person
+* resources
+* mapping to work product
+* start, end, duration, effort
+
+Note: The information on start, end, duration, and effort may sometimes be complicated
+to estimate in the execution in an open source environment. Nevertheless, tasks
+should be planned as part of releases, which sets already an implicit
+duration and end date.
+
+We strongly encourage every software module project lead to use *GitHub
```

Comment:
why only encourage? we are defining the process also for the module projects. Otherwise it may be we are missing content needed from Iso 26262.

Reply:
Fixed